okellogabrielinnocent/fastfoodreact
# feat(signup-login):implement  user authentication

**What does this PR do?**
Implement login functionality for users to be able to login into the fast food fast application.

**Description of tasks to be completed?**

-  implement user signup
-  implement user login
**How can this be manually tested?**

* clone the repository.
* Check out this branch `ft-user-login-163722492`
* Run the app using `yarn start`
* Navigate to the login page on this route `/login`

**What are the related pivotal tracker stories?**
#163818183

**Screenshots?**
